dotcom-rendering/src/layouts/DecideLayout.tsx

```diff
@@ -38,8 +38,13 @@ const DecideLayoutApps = ({ article, format, renderingTarget }: AppProps) => {
 					return notSupported;
 				}
 				default: {
-					// Should be FullPageInteractiveLayout once implemented for apps
-					return notSupported;
+					return (
+						<ImmersiveLayout
+							article={article}
+							format={format}
+							renderingTarget={renderingTarget}
+						/>
```

Comment:
Here I tried to follow [what `DecideLayoutWeb` does](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/layouts/DecideLayout.tsx#L140-L165). I am not sure about line 36-37 above though, I think we should render `ImmersiveLayout` for all designs when `ArticleDisplay.Immersive`. Anyone knows more / the reason why we are saying "Should be InteractiveLayout once implemented for apps"?

Reply:
Oops! That's my mistake when I copied the `DecideLayoutWeb` logic. You're right, this branch should be `ImmersiveLayout`!